share/templates.d/99-generic/live/s390.tmpl

```diff
@@ -6,6 +6,7 @@ KERNELDIR=BOOTDIR
 INITRD_ADDRESS="0x02000000"
 LIVEDIR="LiveOS"
 LORAXDIR="usr/share/lorax/"
+MKS390IMAGE="/usr/share/s390-tools/netboot/mk-s390image"
```

Comment:
This is now `/usr/bin/mk-s390image` since `s390utils-2.15.0-2.fc34`

Reply:
Ok, weird. I thought I pushed that with the requires change. good catch.